bigtable/setup.py

```diff
@@ -30,7 +30,6 @@
 release_status = 'Development Status :: 4 - Beta'
 dependencies = [
     'google-api-core[grpc] >= 1.4.1, < 2.0.0dev',
-    'google-cloud-core >= 0.28.0, <0.29dev',
```

Comment:
I don't think you can drop the 'core' dependency yet:  for instance, the file above is still using `google.cloud._helpers`.

```bash
$ git grep "from google\.cloud import" bigtable/ | grep -v "import bigtable"
bigtable/google/cloud/bigtable/column_family.py:from google.cloud import _helpers
```

Reply:
I sort of suspected so. I was abusing ci a bit. I will revert the changes to setup.py in both for now.